src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityTank.java

```diff
@@ -695,6 +702,9 @@ public NBTTagCompound writeToNBT(NBTTagCompound data) {
     @Override
     public void readFromNBT(NBTTagCompound data) {
         super.readFromNBT(data);
+        if (data.hasKey("blockedSides")) {
+            this.blockedSides = data.getInteger("blockedSides");
+        }
```

Comment:
I think this could be handled by more simple solution. Because `blockedSides` is information that can be found just from this block alone and it's computation is not that expensive so I would rather calculate it on load then save it to NBT